service_mesh/v2x/ossm-reference-jaeger.adoc

```diff
@@ -0,0 +1,34 @@
+[id="jaeger-config-ref"]
+= Jaeger configuration reference
+include::modules/ossm-document-attributes.adoc[]
+:context: jaeger-config-ref
+
+toc::[]
+
+When the {ProductShortName} Operator deploys the `ServiceMeshControlPlane` resource it can also create the resources for distributed tracing. {ProductShortName} uses Jaeger for distributed tracing.
+
+include::modules/ossm-enabling-jaeger.adoc[leveloffset=+1]
+
+include::modules/ossm-configuring-jaeger.adoc[leveloffset=+1]
+
+include::modules/ossm-deploying-jaeger.adoc[leveloffset=+1]
+
+include::modules/ossm-configuring-external-jaeger.adoc[leveloffset=+1]
+
+include::modules/jaeger-deployment-best-practices.adoc[leveloffset=+2]
+
+include::modules/jaeger-config-default.adoc[leveloffset=+2]
+
+include::modules/jaeger-config-collector.adoc[leveloffset=+2]
+
+include::modules/jaeger-config-sampling.adoc[leveloffset=+2]
+
+include::modules/jaeger-config-storage.adoc[leveloffset=+2]
+
+For more information about configuring Elasticsearch with {product-title}, see  xref:../../logging/config/cluster-logging-log-store.adoc[Configuring the log store] or xref:../../jaeger/jaeger_install/rhbjaeger-deploying.adoc[Configuring and deploying Jaeger].
```

Comment:
Are the cross-references verified in some way? `rhbjaeger` seems like an odd (or old) name :-)

Reply:
It is an old name, but if I change the assembly file names we have to create redirects for the URLs in the already published content.